Formula/libbitcoin.rb

```diff
@@ -0,0 +1,59 @@
+class Libbitcoin < Formula
+  desc "Bitcoin Cross-Platform C++ Development Toolkit"
+  homepage "https://libbitcoin.org/"
+  url "https://github.com/libbitcoin/libbitcoin/archive/v3.3.0.tar.gz"
+  sha256 "391913a73615afcb42c6a7c4736f23888cfc999a899fc38395ddcbd560251d94"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+
+  resource "secp256k1" do
+    url "https://github.com/libbitcoin/secp256k1/archive/v0.1.0.13.tar.gz"
+    sha256 "9e48dbc88d0fb5646d40ea12df9375c577f0e77525e49833fb744d3c2a69e727"
+  end
+
+  def install
+    resource("secp256k1").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}",
```

Comment:
Fair enough. I'm able to get the other formulas working by defining secp256k1 as a resource, but those libraries end up linking to the version installed in their own cellar, which is fine.

Unless you have any more change requests, I'm all done! Thanks for the help!